metaworld/envs/mujoco/sawyer_xyz/sawyer_basketball.py

```diff
@@ -6,12 +6,13 @@
 
 
 class SawyerBasketballEnv(SawyerXYZEnv):
+    goal_low = (-0.1, 0.85, 0.15)
```

Comment:
perhaps move these all to module constants, so they are harder to use an import (i.e. they won't be properties on the object when you run `dir(env)`

```py
from metaworld.envs.mujoco.sawyer_xyz.sawyer_basketball import _metaworld_envs_mujoco_sawyer_xyz_sawyer_basketball__GOAL_LOW_DO_NOT_USE   # automatically mangled by the interpreter
```

is a lot harder to discover and use than
```py
> dir(env)
env.goal_low, env.goal_high, etc.
```